test/compflags/coodie/llvmPrintIrBasic.good

```diff
@@ -0,0 +1,13 @@
+; LLVM IR representation of mainTest function after basic optimization stage
+define internal void @mainTest_chpl() {
+entry:
+  %local__str_literal_1924_chpl = alloca %string
+  br label %mainTest_chpl_2blk_body_
+
+mainTest_chpl_2blk_body_:                         ; preds = %entry
+  %0 = load %string, %string* @_str_literal_1924, !tbaa !0
+  store %string %0, %string* %local__str_literal_1924_chpl, !tbaa !0
+  call void @writeln_chpl2(%string* %local__str_literal_1924_chpl, i64 3, i32 51)
```

Comment:
writeln_chpl2 won't be stable as the compiler changes.
Can you put a PREDIFF that just greps out everything except for the "LLVM IR representation" comment?